src/en/components/pagination/design.md

```diff
@@ -6,4 +6,89 @@ tags: ['paginationEN', 'design']
 date: "git Last Modified"
 ---
 
-## Design
\ No newline at end of file
+## Pagination anatomy
+
+### Simple pagination anatomy
+
+#### Previous link
+
+<ol class="anatomy-list">
+  <li>The <strong>previous arrow</strong> is a visual aid to indicate navigating backward, to previous pages in the page flow.</li>
+  <li>The <strong>previous text</strong> is the text that accompanies the previous arrow and indicates that the user would be navigating to the previous page in the flow.</li>
+  <li>The <strong>previous label</strong> indicates the label of the previous page to the user.</li>
+</ol>
+
+<img class="b-sm b-default p-400" src="/images/en/components/anatomy/gcds-pagination-simple-previous-anatomy.svg" alt="The pagination component with numbers pointing to individual parts of the simple pagination's previous link anatomy." />
+
+#### Next link
+
+<ol class="anatomy-list" style="--start-value: 4">
+  <li>The <strong>next text</strong> is the text that accompanies the next arrow and indicates that the user would be navigating to the next page in the flow.</li>
+  <li>The <strong>next arrow</strong> is a visual aid indicating that you would be navigating forwards in the page flow.</li>
+  <li>The <strong>next label</strong> indicates the label of the previous page to the user.</li>
+</ol>
+
+<img class="b-sm b-default p-400" src="/images/en/components/anatomy/gcds-pagination-simple-next-anatomy.svg" alt="The pagination component with numbers pointing to individual parts of the simple pagination's next link anatomy." />
```

Comment:
Looks like you are using the anatomy text + image from the previous link here instead of the next link.